exchange/utils_test.go

```diff
@@ -202,68 +202,76 @@ func TestCleanOpenRTBRequestsCOPPA(t *testing.T) {
 
 func TestCleanOpenRTBRequestsSChain(t *testing.T) {
 	testCases := []struct {
-		description  string
-		inSourceExt  json.RawMessage
-		inExt        json.RawMessage
-		outSourceExt json.RawMessage
-		outExt       json.RawMessage
-		hasError     bool
+		description   string
+		inExt         json.RawMessage
+		inSourceExt   json.RawMessage
+		outSourceExt  json.RawMessage
+		outRequestExt json.RawMessage
+		hasError      bool
 	}{
 		{
-			description:  "Empty root ext and source ext",
-			inSourceExt:  json.RawMessage(``),
-			inExt:        json.RawMessage(``),
-			outSourceExt: json.RawMessage(``),
-			outExt:       json.RawMessage(``),
-			hasError:     false,
+			description:   "Empty root ext and source ext, nil unmarshaled ext",
```

Comment:
what are you testing here with the references to the "unmarshaled ext"? Which input in this test does that refer to?

Reply:
Test was modified. It tries to test the scenario when we pass a `nil` `requestExt *openrtb_ext.ExtRequest` to `cleanOpenRTBRequests`